### PR TITLE
Retrieve harvested data without PR value

### LINKED ISSUE #585: harvested data for curation not showing
If you hit https://clearlydefined.io/curations/1136 and look at the Harvested data section you will see it is empty. Looking at the f12 network trace you can see that in loading the full details view the UI is accessing https://api.clearlydefined.io/harvest/npm/npmjs/-/timers-browserify/2.0.2/pr/1136?form=raw. Note the `/pr/1136` part of the URL. That should not be in there. If you remove those segments the correct result comes back.

### PR DESCRIPTION
**Pull request by storrisi**
Fixes #585

### CODE REVIEW

File: src/utils/entitySpec.js [hunk 1]
```diff
@@ -103,6 +103,10 @@ export default class EntitySpec {
     return new EntitySpec(o.type, o.provider, o.namespace, o.name)
   }
 
+  static asPrLess(o) {
+    return new EntitySpec(o.type, o.provider, o.namespace, o.name, o.revision)
+  }
```
Comment (dabutvin) [inline src/utils/entitySpec.js]: 👍

**Comment by dabutvin** [comment]
@storrisi any idea about the 'current definition' tab?

image

looking at https://deploy-preview-586--clearlydefined.netlify.com/curations/678 it is blank, but the other 2 tabs are working

**Comment by storrisi** [comment]
@dabutvin weird, i will take a look at it, thanks

**Comment by storrisi** [comment]
@dabutvin I've found and fixed the problem.

File: src/utils/entitySpec.js [hunk 2]
```diff
@@ -103,6 +103,10 @@ export default class EntitySpec {
     return new EntitySpec(o.type, o.provider, o.namespace, o.name)
   }
 
+  static asPrLess(o) {
```
Comment (jeffmcaffer) [inline src/utils/entitySpec.js]: `fromObject` would be more generically useful. I'll change that in another PR as this is merged

**Pull request merged**